account_invoice_import_invoice2data/README.rst

```diff
@@ -13,13 +13,13 @@ To know the full story behind the development of this module, read this `blog po
 Installation
 ============
 
-This module requires the Python library *invoice2data* available on `Github <https://github.com/m3nu/invoice2data>`_.
+This module requires the Python library *invoice2data* available on `Github <https://github.com/m3nu/invoice2data>`_ with a version >= 0.2.74 (February 2018).
```

Comment:
Change this in requirements.txt in the repo

Reply:
done